docs/Users_Guide/skew_t.rst

```diff
@@ -0,0 +1,157 @@
+********************
+Skew-T Log-P Diagram
+********************
+
+Description
+===========
+A skew-T log-P plot (skew-T for short) is a thermodynamic diagram used for plotting upper air observations. This skew-T plotting capability was developed to read in and the plot the ASCII output of tropical cyclone diagnostic (TC-Diag) sounding data generated by a diagnostics code set developed by Colorado Institute for Research in the Atmosphere (CIRA). The TC-Diag data are currently derived along a projected or specified track of the center of a tropical cyclone and are meant to represent the thermodynamic environment that the storm will be in if it follows the given track. This TC-Diag code set is in the process of being incorporated into the METplus suite and will be available in an upcoming release. This chapter explains how to create skew-T plots from TC-Diag output using METplotpy's Python-based plotter.
+
+For more information on the skew-T log-P plot, please refer to the following:
+
+https://en.wikipedia.org/wiki/Skew-T_log-P_diagram
+
+https://www.e-education.psu.edu/meteo300/node/638
+
+
+
+Example
+=======
+
+Sample Data
+___________
+
+Sample data from CIRA are included with the source code. The data are ASCII text files in columnar format. The data are located in:
+*$METPLOTPY_BASE/metplotpy/test/skew_t/data*
+
+where *$METPLOTPY_BASE* is the directory where the METplotpy code is saved.
+
+e.g.,
+
+*/usr/path/to/METplotpy*  if the source code was cloned or forked from the Github repository
+
+or
+
+*/usr/path/to/METplotpy-x.y.z*  if the source code was downloaded as a zip or gzip'd tar file from the Release link of
+the Github repository.  The *x.y.z* is the release number.
+
+
+For background information on the CIRA data, please refer to:
```

Comment:
Recommend changing "data" to "TC model diagnostics".

Reply:
Changed CIRA data to CIRA TC model diagnostics.